### DISCUSSION TITLE
Actividad 0 - Bruno, Virginia

### DISCUSSION THREAD

**Discussion by VirBruno** [frlp-utn-ingsoft/catedra#57] [General]
https://github.com/VirBruno/TP0

**Comment by RodrigoJacznik** [comment]
Hola,
buenos commits, la app funciona bien.
Actividad aprobada

**Reply by VirBruno** [reply]
20/4 - Se corrigen errores vistos en clase del 15/4 y se pushean a mismo repo en git
